spec/index.test.js

```diff
@@ -383,7 +383,12 @@ describe('Middleware::Api', () => {
       it('trigger afterError of path2', async () => {
         await apiMiddleware({ dispatch, getState })(next)(action)
         expect(afterError2).toBeCalledWith({
-          getState
+          getState,
+          error: expect.objectContaining({
```

Comment:
how about using a private method to generate the `error`? can be used [here](https://github.com/CodementorIO/redux-api-middleman/pull/16/files#diff-8b9a6468bb0a15a0e2e40a314e776101R413) as well

Reply:
The `error` contains some other http error stuff provided by axios, which we might not need to take care in our tests. So I think it could remain the same in this case.

I'll add test for `error.data`!